java/com/google/flatbuffers/Table.java

```diff
@@ -172,6 +172,30 @@ protected ByteBuffer __vector_as_bytebuffer(int vector_offset, int elem_size) {
     return bb;
   }
 
+  /**
+   * Initialize vector as a ByteBuffer.
+   *
+   * This is more efficient than using duplicate, since it doesn't copy the data
+   * nor allocattes a new {@link ByteBuffer}, creating no garbage to be collected.
+   *
+   * @param bb The {@link ByteBuffer} for the array
+   * @param vector_offset The position of the vector in the byte buffer
+   * @param elem_size The size of each element in the array
+   * @return The {@link ByteBuffer} for the array
+   */
+  protected ByteBuffer __vector_in_bytebuffer(ByteBuffer bb, int vector_offset, int elem_size) {
+    int o = this.__offset(vector_offset);
+    if (o == 0) {
+      bb.position(bb.limit());
```

Comment:
maybe keep the existing behavior of returning `null`, which is easier to test if a field is available or not.